homeassistant/components/mediaroom/media_player.py

```diff
@@ -189,27 +190,31 @@ async def async_notify_received(notify):
             )
         )
 
-    async def async_play_media(self, media_type, media_id, **kwargs):
+    async def async_play_media(
+        self, media_type: str, media_id: str, **kwargs: Any
+    ) -> None:
         """Play media."""
 
         _LOGGER.debug(
             "STB(%s) Play media: %s (%s)", self.stb.stb_ip, media_id, media_type
         )
+        command: str | int
         if media_type == MEDIA_TYPE_CHANNEL:
             if not media_id.isdigit():
                 _LOGGER.error("Invalid media_id %s: Must be a channel number", media_id)
                 return
-            media_id = int(media_id)
+            command = int(media_id)
```

Comment:
`mypy` does not like to set `media_id` to an `int`.
I think it's clearer to create an intermediate variable here.